packages/styles/components/_c.notification.scss

```diff
@@ -0,0 +1,110 @@
+$text-font-size: get-token(size, font, '04');
+$title-font-size: get-token(size, font, '05');
+
+.mc-notification {
+  box-sizing: border-box;
+  position: relative;
+  display: inline-block;
```

Comment:
I think display block or even flex (without relative/absolute) positioning would be better.

Reply:
the notification should take all the available width.